projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/payment-method/payment-form/payment-form.component.html

```diff
@@ -95,7 +95,6 @@
                 [type]="iconTypes.INFO"
                 class="cx-payment-form-tooltip"
                 placement="right"
-                ngbTooltip="Card Verification Value"
```

Comment:
I don't think so just removal of this toolip is good idea. Let's use title="Card Verification Value" here

Reply:
the ticket clearly states what should be done ;) I will replace tooltip with title attribute anyways